examples/auth/auth/index.js

```diff
@@ -0,0 +1,46 @@
+import { Strategy as strategy } from 'passport-local'
+import jwt from 'jsonwebtoken'
+import uuid from 'uuid'
+
+const options = {
+  strategy: {
+    usernameField: 'username',
+    passwordField: 'username',
+    successRedirect: null
+  },
+  routes: {
+    register: {
+      path: '/register',
+      method: 'POST'
+    },
+    login: {
+      path: '/login',
+      method: 'POST'
+    }
+  },
+  registerCallback: async (_, username) => {
+    return jwt.sign(
+      {
+        name: username,
+        id: uuid.v4()
+      },
+      'SECRETJWT'
+    ) // TODO - hide jwt Secret
+  },
+  loginCallback: async (_, username) => {
+    return jwt.sign(
+      {
+        name: username,
+        id: uuid.v4()
```

Comment:
Identity even after success login becomes new guid random value, is it expected behavior?

Reply:
May be in this case id is not needed. I'll remove it, thx.